docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3'
 services:
   testnet_node_alice:
     container_name: testnet_node_alice
-    image: cennznet/cennznet:1.0.0-rc2
+    image: cennznet/cennznet:latest
```

Comment:
Would it be better to stay with rc2, only if cennznet:latest is always stable, otherwise, api automation tests may be failed? @holygits

Reply:
we really should tag with a commit as well in cennznet but for now workaround is either use sha hash of image so it won't update: `cennznet/cennznet@sha256:8a84ab9366c3be3c5d2062b5df52969179840addfd58f289608bfc612754b6c3` or I don' think circleci will re-pull the latest image anyway

Reply:
added Sha hash of image